.npmignore

```diff
@@ -1 +1,2 @@
 *~
+package-lock.json
```

Comment:
no.

Reply:
Done

Reply:
We need a .npmrc file to disable package lock.

Reply:
done